src/main/java/io/minio/MinioClient.java

```diff
@@ -330,23 +330,23 @@ public MinioClient(String endpoint, int port, String accessKey, String secretKey
    *
    * @param accessKey Access key to access service in endpoint.
    * @param secretKey Secret key to access service in endpoint.
-   * @param insecure  If true, access endpoint using HTTP else access it using HTTPS.
+   * @param secure If false, access endpoint using HTTP else access it using HTTPS.
```

Comment:
Please reword to have positive case first like
"If true, access endpoint using HTTPS else access it using HTTP"